CGPs/cgp-0060.md

```diff
@@ -256,6 +267,11 @@ The setup of the liquidity pool is based on the assumption that the cUSD/USDC pr
 
 The cUSD/USDCet pool will offer cUSD (and USDCet) at an incorrect price that is either higher (cUSD/USDCet < 1) or lower (cUSD/USDCet > 1) than the market price. The availability of expensive/cheap liquidity can have a price impact that is undesirable for cUSD as it is purely caused by a USDCet price deviation.
 
+ 
+## Market Interference
+Whenever arbitrageurs have matched the StableSwap cUSD/USDCe price with the price on other pools, most users will trade with the reserve StableSwap pool since it has lower fees and slippage especially for larger trades. If DEX routers include the StableSwap pool, this shifts trading volume towards the StableSwap pool.
```

Comment:
```suggestion
Whenever arbitrageurs have matched the the prices on other pools with the StableSwap cUSD/USDCe price, most users will trade with the reserve StableSwap pool since it has lower fees and slippage especially for larger trades.
```